src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java

```diff
@@ -110,4 +111,10 @@ public WelshLanguageRequirements getWelshLanguageRequirements() {
     public StatementOfTruth getStatementOfTruth() {
         return respondent1DQStatementOfTruth;
     }
+
+    @Override
```

Comment:
Please could we try removing 'Override', which probably forces us to put the signature in interface DQ.java, and hence in Applicant1DQ.java

Reply:
Done